react/src/components/CommentaryList/index.js

```diff
@@ -1,24 +1,28 @@
 import React, { Component } from 'react';
+import Moment from 'react-moment';
 
 import UserAvatar from '../UserAvatar'
 
 import './styles.css';
 
 class CommentaryList extends Component {
   render() {
-    const commentaryList = this.props.commentaries.map((commentary, i) => {
-      return (
-        <div className="commentary" key={i}>
-          <UserAvatar />
-          <h3 className="commentary-title">{commentary.name}</h3>
-          <h4 className="commentary-date">{commentary.date}</h4>
-          <p className="commentary-text">{commentary.text}</p>
-        </div>
-      );
-    });
     return (
       <div>
-        {commentaryList}
+        {
+          this.props.commentaries.sort((a, b) => b.id-a.id).slice(0, this.props.max).map((commentary, i) => {
```

Comment:
This component is manipulating the data and rendering it. So, this component is doing more than one thing. Would you like to do this sort at other level ? It could be:
* At Container Component level or
* At reducer level